src/components/refactor/dialogs/two-windings-transformer-creation/two-windings-transformer-creation-dialog.js

```diff
@@ -575,6 +619,9 @@ const TwoWindingsTransformerCreationDialog = ({
                     ...twt[PHASE_TAP_CHANGER],
                 };
             }
+
+            console.log(currentLimits1);
+            console.log(currentLimits2);
```

Comment:
I think those should be removed

Reply:
Done, there's always one or two debug message that slip through the net..